test/jdk/java/nio/channels/etc/MemorySegments.java

```diff
@@ -467,6 +650,129 @@ void testFileChannelScatteringRead(Supplier<Arena> arenaSupplier) throws IOExcep
         }
     }
 
+    /**
+     * AsynchronousFileChannel read/write(ByteBuffer).
+     */
+    @DisabledOnOs(OS.WINDOWS)
+    @ParameterizedTest
+    @MethodSource("arenaSuppliers")
+    void testAsyncFileChannelReadWrite(Supplier<Arena> arenaSupplier) throws Throwable {
+        boolean confined = isConfined(arenaSupplier);
+        Arena arena = arenaSupplier.get();
+
+        Path file = Files.createTempFile(Path.of(""), "foo", ".dat");
```

Comment:
Is the use of an empty `Path` instead of `Path.of(".")` intentional here? The documentation of `Path` states:


> Accessing a file using an <i>empty path</i> is equivalent to accessing the default directory of the file system.

Unlike the current working directory, I couldn't find the definition of default directory. Are they the same for the default filesystem?